server/py/services/api/tests/unit/crud/runtimes/nuclio/test_helpers.py

```diff
@@ -106,15 +106,15 @@ def test_resolve_work_dir_and_handler(handler, expected):
 @pytest.mark.parametrize(
     "mlrun_client_version,python_version,expected_runtime",
     [
-        ("1.3.0", "3.9.16", "python:3.9"),
-        ("1.3.0", "3.7.16", "python:3.7"),
+        ("1.9.0", "3.12.16", mlrun.mlconf.default_nuclio_runtime),
```

Comment:
shouldn't the expected be python 3.12?

Reply:
It should, in the next version 😉 
This case doesn't go into [this condition](https://github.com/mlrun/mlrun/pull/7336/files#diff-c63a1c7dee3768ff7741fb563d0813efa6dda440864fc7d375036c136e1d651aR61-R65), so it returns the default version.
When the default will be 3.12, it will return it properly.